bundles/org.openhab.binding.nikohomecontrol/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -186,20 +179,159 @@
 			</parameter>
 		</config-description>
 	</thing-type>
+	<thing-type id="energyMeterLive">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge"/>
+		</supported-bridge-type-refs>
+		<label>@text/energyMeterLiveLabel</label>
+		<description>@text/energyMeterLiveDescription</description>
+		<channels>
+			<channel id="power" typeId="power"/>
+			<channel id="energy" typeId="energy"/>
+			<channel id="energyday" typeId="energy"/>
+			<channel id="energylast" typeId="measurementtime"/>
+		</channels>
+		<config-description>
+			<parameter name="energyMeterId" type="text" required="true">
```

Comment:
Isn't it rather `meterId` now ?

Reply:
I intended to keep it for backward compatibility, but it is probably better to just change it. There aren't many users for energy meters, so the impact shouldn't be that big.